index.html

```diff
@@ -0,0 +1,462 @@
+            <div class="card-text">
+              <h2><b><i>INTERNATIONAL RESEARCH</i></b></h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <div class="card text-center mb-3" style="box-shadow:none; min-height:20px;">
+            <img src="http://technonjr.org/wp-content/uploads/2015/03/website-poster-1024x576.jpg" class="card-image-top" alt="International Research">
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <a href="http://technonjr.org/department/departments/research/" class="btn btn-lg btn-secondary">READ MORE</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!--Departments-->
+
+<section id="departments" style="padding-top:10%;">
+  <div class="container">
+    <div class="title">
+      <h1>DEPARTMENTS</h1>
+      <hr>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img class="details card-image-top" src="http://technonjr.org/wp-content/uploads/2014/10/computer-science-550x400.jpg" alt="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)">
+          <div class="image__overlay image__overlay--blur">
+            <p class="image__description">
+              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="http://technonjr.org/department/cse/ ">
+                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
+              </a>
+            </p>
+          </div>
+          <div class="card-body">
+            <h5 class="card-text">
+              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="http://technonjr.org/department/cse/ ">
+                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
+              </a>
+            </h5>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img src="http://technonjr.org/wp-content/uploads/2014/10/electricals-550x400.jpg" class="details card-image-top" alt="Electronics and Communication Engineering (ECE)">
+          <div class="image__overlay image__overlay--blur">
+            <p class="image__description">
+              <a title="Electronics and Communication Engineering (ECE)" href="http://technonjr.org/department/electronics-and-communication-engineering-ece/ ">
+                Electronics and Communication Engineering (ECE)
+              </a>
+            </p>
+          </div>
+          <div class="card-body">
+            <h5 class="card-text">
+              <a title="Electronics and Communication Engineering (ECE)" href="http://technonjr.org/department/electronics-and-communication-engineering-ece/ ">
```

Comment:
```suggestion
              <a title="Electronics and Communication Engineering (ECE)" href="">
```
Remove link to external website